docs/command_line_tool/phylum.md

```diff
@@ -4,37 +4,51 @@ category: 6255e67693d5200013b1fa3e
 hidden: false
 ---
 
-Interact with the Phylum API
-
-### Core Commands
-* [phylum analyze](https://docs.phylum.io/docs/phylum_analyze)
-* [phylum auth](https://docs.phylum.io/docs/phylum_auth)
-* [phylum project](https://docs.phylum.io/docs/phylum_project)
-
-### Additional Commands
-* [phylum group](https://docs.phylum.io/docs/phylum_group)
-* [phylum help](https://docs.phylum.io/docs/phylum_help)
-* [phylum history](https://docs.phylum.io/docs/phylum_history)
-* [phylum package](https://docs.phylum.io/docs/phylum_package)
-* [phylum parse](https://docs.phylum.io/docs/phylum_parse)
-* [phylum ping](https://docs.phylum.io/docs/phylum_ping)
-* [phylum uninstall](https://docs.phylum.io/docs/phylum_uninstall)
-* [phylum update](https://docs.phylum.io/docs/phylum_update)
-* [phylum version](https://docs.phylum.io/docs/phylum_version)
+Client interface to the Phylum system
+
+```sh
+Usage: phylum [OPTIONS] [COMMAND]
+```
 
 ### Options
-  `-c`, `--config <FILE>`
-  &emsp; Use a specific configuration file
-  
-  `-h`, `--help`
-  &emsp; Print help information
-  
-  `--no-check-certificate`
-  &emsp; Don't validate the server certificate when performing API requests
-  &emsp; DANGER: this should only be used when SSL termination breaks the Phylum certificate pinning
```

Comment:
This line got dropped but appears important enough that is should continue to be included somehow.

Reply:
Considering it's not in the CLI either, I'm not sure it's worth it tbh.

Not checking certificates should be pretty obviously dangerous to any user, so I'm not sure if this comment provides actual value?

Reply:
I like having this kind of information in the CLI help output. It makes that output more verbose, but that is the place I (and I assume most users) will look first for "documentation." My guess is this language was initially added b/c someone was (ab)using this feature the wrong way or otherwise had a question about the security implications.